zookeeper-api/src/main/java/org/apache/helix/zookeeper/zkclient/callback/ZkAsyncCallbacks.java

```diff
@@ -111,12 +112,17 @@ protected void recordFailure(int rc, String path, ZkAsyncCallMonitorContext moni
     }
   }
 
-  public static class CreateCallbackHandler extends DefaultCallback implements StringCallback {
+  public static class CreateCallbackHandler extends DefaultCallback implements StringCallback, Create2Callback {
     @Override
     public void processResult(int rc, String path, Object ctx, String name) {
       callback(rc, path, ctx);
     }
 
+    @Override
+    public void processResult(int rc, String path, Object ctx, String name, Stat stat) {
```

Comment:
what is 'stat' object used for??

Reply:
It's a Zookeeper data structure used for returning metadata about a ZNode. This particular method is part of the `Create2Callback` interface that has to be implemented.